community/kernel/src/main/java/org/neo4j/kernel/impl/api/store/StorePropertyCursor.java

```diff
@@ -302,77 +279,64 @@ private boolean nextBlock()
         // Read remaining data from previous property (if it was not read)
         while ( remainingBlocksToRead-- > 0 )
         {
+            // don't really care to retry since we discard the data...
```

Comment:
those are values of a property block that you are not interested in, that's why we are skipping them. E.g., this might happen when you do:
```
cursor.next();
cursor.next();
```